src/main/resources/db/changelog/changes/16-update-orders-table.yaml

```diff
@@ -0,0 +1,9 @@
+databaseChangeLog:
+  - changeSet:
+      id: 16
+      author: ChorniyPistolet
+      changes:
+        - modifyDataType:
```

Comment:
remove this changeSet and id: 17 also
modify datatype columnName: status in existing changesets